src/pages/Main/Tealist/Detail/Selectlist/Selectlist.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Selectlist.scss';
+
+class Selectlist extends React.Component {
+  state = {
+    isClicked: false,
+    currentId: 1,
+  };
+
+  selectShoppingBag = id => {
+    this.toggleOptions();
+
+    this.setState({ currentId: id }, () => {
+      this.props.includeBagPrice(this.state.currentId);
+    });
```

Comment:
인자로 받은 id값에 따라서 둘다 바꾸는 거니까 굳이 콜백으로 순차적으로 실행되게 할 필요 없어보입니다!
```suggestion
    this.setState({ currentId: id });
    this.props.includeBagPrice(id);
```

Reply:
저렇게 코드를 작성하니 한 박자 늦게 실행되어서 완전히 업데이트 한 뒤에 실행되도록 콜백 한 건데..지금 다시 수정해보니 정상적으로 적용되네요..뭘까요...

Reply:
아마도 includBagPrice에 인자로 전달하는 부분을 인자로받은 id가 아니라 this.state.currentId를 인자로 전달하셔서 한박자 늦게 실행되셨던 것 같습니다!
지금은 state값을 인자로 사용하는게 아니라 인자로받은 id값을 그대로 다시 인자로 넣어서 호출하고 있어서 이상없게 동작하는걸꺼에요!

Reply:
아 그렇군요! 알려주셔서 감사합니다!